azavea/azavea-branding-guide
# Add product logo packages

## Overview
We had added .zip folders of the Azavea logo at various sizes and for print/web to the branding guide upon launch. We had not yet wrangled all of our product, open source, fellowship, and event logos, however. This PR adds a .zip and button to each.

### Demo
<img width="1679" alt="fellowships___azavea_branding_guide" src="https://user-images.githubusercontent.com/5672295/46766514-3462ca00-ccb0-11e8-860a-7c0fbd28d3aa.png">

<img width="882" alt="screen shot 2018-10-10 at 5 15 38 pm" src="https://user-images.githubusercontent.com/5672295/46766493-23b25400-ccb0-11e8-9fb4-496429da9b77.png">


### Notes
- I also added a release note for this addition
- As stated in the release note, the exact contents of each differs slightly, based on what was originally created for each logo (some don't have a white version, some don't have a "light" version, etc). Sizes of .png files are all the same, however.


## Testing Instructions
* `git pull`
* Go to Identity > Products
* Confirm that each download button works and contains what is expected
* Go to Identity > Open Source
* Confirm that each download button works and contains what is expected
* Go to Identity > Fellowships
* Confirm that each download button works and contains what is expected
* Go to Identity > Community
* Confirm that each download button works and contains what is expected

Closes #100
